src/fr/mrc/ptichat/connection/Input.java

```diff
@@ -0,0 +1,15 @@
+package fr.mrc.ptichat.connection;
+
+import java.util.Scanner;
+
+public class Input {
```

Comment:
Useful class for the moment, but we have to see how to link the ui part to your code